fe/fe-core/src/main/java/org/apache/doris/planner/OriginalPlanner.java

```diff
@@ -167,7 +167,9 @@ public void createPlanFragments(StatementBase statement, Analyzer analyzer, TQue
         PlanNode singleNodePlan = singleNodePlanner.createSingleNodePlan();
```

Comment:
Is it better to move setQueryJoinReorderFinishTime() here?

Reply:
done